week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,33 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
+  fillUp: function () {
+    return (this.volume = this.volume + 100);

```

Comment:
tow issues here
big one, the overall volume should not exceed 100 ml, in the way you wrote it may go over 100 ml!
small one, do you need the return ?